internal/app/machined/pkg/runtime/v1alpha1/bootloader/sdboot/sdboot.go

```diff
@@ -216,6 +218,29 @@ func (c *Config) KexecLoad(r runtime.Runtime, disk string) error {
 			return fmt.Errorf("failed to read cmdline from uki: %w", err)
 		}
 
+		smbiosInfo, err := smbiosinternal.GetSMBIOSInfo()
```

Comment:
I know we don't support kexec today with SecureBoot, but I think it'd be nice still to add here `if !SecureBoot`

Reply:
this doesn't get called at all if secureboot is enabled, it shortcuts way before in sequencer